acpi/src/fadt.rs

```diff
@@ -340,11 +338,172 @@ impl Fadt {
 }
 
 #[derive(Clone, Copy)]
-// TODO: methods for other flags
-pub struct Flags(u32);
+pub struct FixedFeatureFlags(u32);
 
-impl Flags {
+impl FixedFeatureFlags {
+    /// If true, an equivalent to the x86 [WBINVD](https://www.felixcloutier.com/x86/wbinvd) instruction is supported.
+    /// All caches will be flushed and invalidated upon completion of this instruction,
+    /// and memory coherency is properly maintained. The cache *SHALL* only contain what OSPM references or allows to be cached.
+    pub fn supports_equivalent_to_wbinvd(&self) -> bool {
+        self.0.get_bit(0)
+    }
+
+    /// If true, [WBINVD](https://www.felixcloutier.com/x86/wbinvd) properly flushes all caches, but memory coherency is not maintained.
+    pub fn wbinvd_flushes_all_caches(&self) -> bool {
+        self.0.get_bit(1)
+    }
+
+    /// If true, all processors implement the C1 power state.
+    pub fn all_procs_support_c1_power_state(&self) -> bool {
+        self.0.get_bit(2)
+    }
+
+    /// If true, the C2 power state is configured to work on a uniprocessor and multiprocessor system.
+    pub fn c2_configured_for_mp_system(&self) -> bool {
+        self.0.get_bit(3)
+    }
+
+    /// If true, the power button is handled as a control method device.
+    /// If false, the power button is handled as a fixed-feature programming model.
+    pub fn power_button_is_control_method(&self) -> bool {
+        self.0.get_bit(4)
+    }
+
+    /// If true, the sleep button is handled as a control method device.
+    /// If false, the sleep button is handled as a fixed-feature programming model.
+    pub fn sleep_button_is_control_method(&self) -> bool {
+        self.0.get_bit(5)
+    }
+
+    /// If true, the RTC wake status is supported in fixed register space.
```

Comment:
Wrong way round - a one indicates that RTC wake status is *not* supported in fixed register space. Function could be called `no_rtc_wake_in_fixed_register_space`?